infra/bicep/main.bicep

```diff
@@ -1,19 +1,19 @@
 param cognitiveCommunicationLocation string
 param embeddingDeploymentType string = 'Standard'  // Pay-as-you-go in a single region
 param embeddingModel string = 'text-embedding-ada-002'
-param embeddingQuota int = 100
+param embeddingQuota int = 10
 param embeddingVersion string = '2'
 param functionappLocation string
-param instance string = deployment().name
+param instance string
```

Comment:
Quota of 10 is far too low to use the system. A single chat request can take up to 20-30k tokens. What are you trying to solve?